### ISSUE TITLE
Support Dropbox app containers

### ISSUE THREAD

**Issue by scaine** [perfectapi/CKP#63]
I would love to use CPK's Dropbox integration, but it's currently implemented in a way that allows CPK access to all my Dropbox files, which isn't ideal.

Dropbox's API supports app-specific containers (https://www.dropbox.com/help/6231) which would mean that CPK only has access to the files it creates. That would allow me to upload my database to CPK's own area, confident that the app can't interfere with the rest of my files.

I've gotten this working pretty easily via the API with pre-created apps via (https://www.dropbox.com/developers/apps), but I think it's reasonably straightforward to use the API to request the creation of an app too. That would happen just after authorising the login.

**Comment by bes** [comment]
Please keep the "any file support" as well, since there might be users (me ;]) who already have to use this feature for another app (Keepass2Android) - otherwise I won't be able to keep the files in sync automatically...

**Comment by DukeyToo** [comment]
I agree, I think this would be best as a separate feature; not a replacement for the current Dropbox functionality.
